Project/src/main/java/com/github/lgooddatepicker/components/CalendarPanel.java

```diff
@@ -940,7 +940,8 @@ private void labelIndicatorMouseEntered(MouseEvent e) {
             return;
         }
         // Highlight the label.
-        label.setBackground(new Color(184, 207, 229));
+        label.setBackground(settings.getColor(DateArea.BackgroundHoverLabel));
+        label.setForeground(settings.getColor(DateArea.TextHoverLabel));
```

Comment:
Why is the *Foreground* set here additionally, if only the *Background* color is now customizeable? Is something missing in the code or am I missing something?

Reply:
Please disregard my precious comment `setForground` means setting the text color in this context, which is correct.